src/js/common/schemaform/review/StringField.jsx

```diff
@@ -14,13 +14,15 @@ import {
  */
 export default function StringField(props) {
   const { registry, schema, uiSchema, formData } = props;
-  const enumOptions = Array.isArray(schema.enum) && optionsList(schema);
+  const uiOptions = getUiOptions(uiSchema);
+  const labels = uiOptions.labels || {};
+  const enumOptions = Array.isArray(schema.enum) && optionsList(
```

Comment:
I think if we're going to override `enumOptions` in here we should remove the code in review/widgets.jsx for `SelectWidget` that checks for `labels` since it won't ever be different from what's in `enumOptions`.

Alternatively, it looks like the bug in the original code is that the `options` prop passed to `Widget` needs to include `labels` as well as `enumOptions`. So I think passing that would also fix this (instead of overriding `enumOptions`).